packages/terra-action-header/package.json

```diff
@@ -21,8 +21,8 @@
   },
   "homepage": "https://github.com/cerner/terra-core#readme",
   "peerDependencies": {
-    "react": "^16.4.2",
-    "react-dom": "^16.4.2"
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5"
```

Comment:
Would it be possible to move these changes to a different PR?

Reply:
👍I've split off the react/react-dom changes into a separate PR here: https://github.com/cerner/terra-core/pull/2298